models/auth/oauth2.go

```diff
@@ -46,6 +47,26 @@ func init() {
 	db.RegisterModel(new(OAuth2Grant))
 }
 
+func Init(ctx context.Context) error {
+	if setting.OAuth2.GitCredentialHelpers {
+		// the following Git credential helpers are universally useful
+		// https://git-scm.com/doc/credential-helpers
+		_ = db.Insert(ctx, []OAuth2Application{
```

Comment:
how about add a subcommand like `./gitea manager init-git-credential` to insert this record into db?